Dockerfile.debian

```diff
@@ -68,6 +68,15 @@ RUN cd /tmp && \
     curl -Lo direnv https://github.com/direnv/direnv/releases/download/v2.15.2/direnv.linux-amd64 && \
     chmod +x direnv && mv direnv /usr/local/bin
 
+# docker
+RUN curl -fsSL https://download.docker.com/linux/debian/gpg | sudo apt-key add - && \
```

Comment:
MAY: consolidate with other apt package installs above to minimize layer

Reply:
Will leave alone.  I like how the docker install stuff is all in its own layer, and grouped together.